app/assets/stylesheets/layouts/starrylight.scss

```diff
@@ -1,86 +1,222 @@
 @import 'variables';
+
+/* - header */
+$bg_color_header_top: transparent; // top header strip for logo
+$bg_color_header_middle: #41416E; // middle header strip with user items
+$bg_color_header_bottom: #6F658D; // bottom header strip with navigation
+
 #header {
-  background-color: #6F658D;
+  background-color: $bg_color_header_bottom;
   background-image: url(image-path("layouts/starry.jpg"));
   background-position: right top;
 }
-#logo, #nav-bottom { background-color: transparent !important; }
-#user-info, #nav-top { background-color: #41416E; background-color: rgba(65, 65, 110, 0.9); }
-
-body { background-color: #B0AFB7; }
-
-th, .content-header { background-color: #42416E; }
-.sub { color: #F3F3F3; }
-.sub, .post-navheader, .flash.breadcrumbs { background-color: #8287A2; }
-.selector-popup, .single-description, .subber, .form-table-ender { background-color: #6F658D; color: #DADADA; }
-th, .single-description, .content-header {
-  a { color: #F3F3F3; }
-  a:visited { color: #C6C6E5; }
+#logo { background-color: $bg_color_header_top !important; }
+#user-info, #nav-top { background-color: $bg_color_header_middle; background-color: rgba(65, 65, 110, 0.9); }
+
+/* - links */
+$font_color_link: #666284; // link
+$font_color_link_visited: #C6C6E5;
+
+a { color: $font_color_link; }
+a:visited { color: #484357; } //TODO: inconsistency between this a:visited and others, ???? this was previous at the bottom of the documment, was it overriding?
```

Comment:
Could you fix this like you did in starry? I expect it works the same. https://github.com/Marri/glowfic/pull/591/commits/41a418251033a48801f0b92659ebaeae1bfe43b9